pkg/services/app/unpause.go

```diff
@@ -0,0 +1,45 @@
+package app
+
+import (
+	"context"
+	"fmt"
+)
+
+type UnpauseParams struct {
+	Name      string
+	Namespace string
+}
+
+func (a *App) Unpause(params UnpauseParams) error {
```

Comment:
The `Pause` and `Unpause` implementations are almost identical. Would it be possible to combine the logic into one function, and pass in the desired operation (`pause` or `unpause`) as an argument?

You could still keep the exported function separate, but they would be thin wrappers around a single `pauseOrUnpause` function that decides which flux operation to do.